sw/device/lib/testing/test_status.c

```diff
@@ -0,0 +1,35 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/test_status.h"
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/log.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/runtime/hart.h"
+
+/**
+ * Address of the memory location to write the test status. For DV use only.
+ */
+static const uintptr_t kSwDvTestStatusAddr = 0x1000fff8;
+
+inline void set_test_status(const test_status_t test_status) {
```

Comment:
Marking a function in a `.c` file `inline` is meaningless. The compiler is unlikely to inline this function.